### ISSUE TITLE
DatabaseConfig via Global Lookup uses deprecated method

### ISSUE THREAD

**Issue by kubukoz** [playframework/play-slick#360]
https://www.playframework.com/documentation/2.5.x/PlaySlick

In the line:

``` scala
val dbConfig = DatabaseConfigProvider.get[JdbcProfile](Play.current)
```

`Play.current` is deprecated, so I think the whole example should be removed (or at least marked as unsupported as of Play 2.5.x). Probably the same goes for the sample project mentioned in this section.

**Comment by jbdeboer** [comment]
I took a crack at a "legacy" pattern in this PR: https://github.com/pedrorijo91/play-slick3-steps/pull/2/files#diff-19895978d4b5a42bd6492df2f2d1c2ceR51

IIUC, play-slick should update the DatabaseConfigProvider to take a Configuration instead of an Application.

**Comment by marcospereira** [comment]
@kubukoz yes, the docs are outdated and you should now just inject a `DatabaseConfigProvider` as you can see here:

https://github.com/playframework/play-slick/blob/master/samples/basic/app/dao/CatDAO.scala

Do you want to submit a pull request to fix this?

**Comment by kubukoz** [comment]
I think @jbdeboer already did a great job at that ;)

**Comment by marcospereira** [comment]
Fixed by #387.
